jobs/harbor/templates/bin/pre-start.erb

```diff
@@ -166,5 +166,13 @@ fi
 #Save installed harbor version to file
 echo $HARBOR_FULL_VERSION > $HARBOR_VERSION_FILE
 
+<%- if p("enable_externalssl") -%>
+################# copy nginx.conf file
+/bin/cp ${HARBOR_JOB_DIR}/config/nginx.conf ${HARBOR_PACKAGE_DIR}/common/config/nginx/nginx.conf
+/bin/sed -i 's/^http:/http:\n  relativeurls: true/' /var/vcap/data/packages/harbor-app/*/common/config/registry/config.yml
```

Comment:
Can we use the path ${HARBOR_PACKAGE_DIR}/common/config/registry/config.yml instead ? I'm not sure if "/var/vcap/data/packages/harbor-app/*" is proper, since it points to many dirs in /var/vcap/data/packages/harbor-app/.

Reply:
Use sed command in config file will bring more issue to reconfigure, for example, I need to set relativeurls: false